config/config.go

```diff
@@ -10,13 +10,15 @@ type Config struct {
 	RendererURL         string
 	FilterAPIURL        string
 	DatasetAPIURL       string
-	CodeListAPIURL      string
 	HierarchyAPIURL     string
 	DatasetAPIAuthToken string
 	FilterAPIAuthToken  string
 	SearchAPIAuthToken  string
+	ServiceAuthToken    string
+	DownloadAuthToken   string
```

Comment:
I don't think this app needs the `DownloadAuthToken` as I believe this is used as a super hacky way to have the download service get a different version of the response for dataset downloads from the dataset API.  Therefore this should only be set in the download service.  I think everywhere else this can be explicitly passed to the client library as `nil`.